README.md

```diff
@@ -24,3 +24,13 @@ Do not modify the `relay/` or `system-parachains/` folder. They are currently re
     └── collectives
         └── collectives-polkadot
 ```
+
+## Approval rights
+
+The approval rights are configured in [`review-bot.yml`](.github/review-bot.yml). The rights are configured as:
+
+- All files in `.github` require two approvals from fellow members of rank 4 or higher.
+- `CHANGELOG.md`, `relay/*` or `system-parachains/*` require four approvals from fellow members of rank 3 or higher.
+- All other files require the approval from one fellow member of rank 2 or higher.
```

Comment:
Technically, a "Fellow" is _member of the Fellowship_ rank 3 or higher. "Rank 2 Fellow" doesn't make sense, they are "Members".
```suggestion
- All files in `.github` require two approvals from Fellowship members of rank 4 or higher.
- `CHANGELOG.md`, `relay/*` or `system-parachains/*` require four approvals from Fellowship members of rank 3 or higher.
- All other files require the approval from one Fellowship member of rank 2 or higher.
```